addon/components/sl-select.js

```diff
@@ -267,16 +267,6 @@ export default Ember.Component.extend( InputBased, TooltipEnabled, {
                 this.set( 'value', input.select2( 'val' ) );
             });
 
-            const originalBodyOverflow = document.body.style.overflow || 'auto';
```

Comment:
The removal of this block of code fixes the shifting issue, where the whole page would shift to one side. https://huboard.com/softlayer/sl-ember-components/#/issues/93135058 referenced in this issue.